mautrix_telegram/__main__.py

```diff
@@ -162,6 +165,13 @@ def is_bridge_ghost(self, user_id: UserID) -> bool:
     async def count_logged_in_users(self) -> int:
         return len([user for user in User.by_tgid.values() if user.tgid])
 
+    def update_bridge_readiness(self):
```

Comment:
Good start, but I think we can make this even cheaper without a loop. We know that `bridge_telegram_update_count` aka `UPDATE_TIME` in `abstract_user.py` effectively flatlines when the bridge is unhappy. So:

I would add a line in https://github.com/vector-im/mautrix-telegram/blob/2a60fdb20b44636bd95b5220e36543de7c320e97/mautrix_telegram/abstract_user.py#L203 that calls `self.bridge.update_bridge_readiness` where it just sets the current timestamp in a value somewhere on the bridge. 

This has a problem though: We need `self.az.ready` to be set to false after two minutes have passed without any activity, which means we need some kind of background task to check the TS and update only then.

We might need something akin to `_loop_active_puppet_metric` but for this, wdyt?

Reply:
I think I've done this. Please review.